reference/commands/creator/new.rst

```diff
@@ -83,3 +85,14 @@ Creates a new package recipe template with a 'conanfile.py' and optionally,
   .. code-block:: bash
 
       $ conan new mypackage/1.0@myuser/stable -t -ciglg -ciglc -ciu https://api.bintray.com/conan/myuser/myrepo
+
+- Create files from a custom, predefined user template:
+
+  .. code-block:: bash
+
+      $ conan new mypackage/1.0 --file=templates/myconanfile.py
```

Comment:
it's not clear how Conan will manage the name and version in the recipe. Maybe a small example here would be fine:

```python
class ConanTemplate(Conanfile):
    name = {{ name }}
    version = {{ version }}
...
```

Reply:
Sure, added it